inc/search.class.php

```diff
@@ -4798,8 +4977,10 @@ static function addDefaultJoin($itemtype, $ref_table, array &$already_link_table
     * @param string  $field                Field to display (needed for translation join) (default '')
     *
     * @return string Left join string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addLeftJoin($itemtype, $ref_table, array &$already_link_tables, $new_table,
+   public function addLeftJoin($itemtype, $ref_table, array &$already_link_tables, $new_table,
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.